modules/ts/src/ts_perf.cpp

```diff
@@ -2207,7 +2207,7 @@ void PrintTo(const MatType& t, ::std::ostream* os)
         case CV_32S: *os << "32S"; break;
         case CV_32F: *os << "32F"; break;
         case CV_64F: *os << "64F"; break;
-        case CV_USRTYPE1: *os << "USRTYPE1"; break;
+        case CV_USRTYPE1: *os << "16F"; break;
         default: *os << "INVALID_TYPE"; break;
```

Comment:
It is time to replace this switch to something of:
```
depthToString(t)
typeToString(t)
```

Reply:
the naive change to typeToString(t) breaks the perf tests because of different naming conventions. Let's leave it as-is for now